.github/workflows/ci.yaml

```diff
@@ -20,10 +20,19 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
-      - name: Set up JDK
+      - name: Set up JDK ${{ matrix.java }}
         uses: actions/setup-java@v3
         with:
           java-version: ${{ matrix.java }}
           distribution: ${{ matrix.distribution }}
-      - name: Test with Maven
+      - name: Test with Maven Using Project Maven Wrapper
         run: ./mvnw clean install site -V -B -D"maven.artifact.threads=64" -D"org.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn"
+      - name: Set up JDK 8 Integration Run (Consumer Run)
```

Comment:
My understanding is that the second task will build and not re-use the previously workspace from the first task right ?

So if we have this additionnel task that is building and running with mv n3.2.5 / java 8, why do we need to keep the jdl 8 in the matrix above ?